.github/workflows/ci.yml

```diff
@@ -6,7 +6,7 @@ jobs:
   test:
     strategy:
       matrix:
-          runner: ["ubuntu-latest", "ubuntu-24.04-arm"]
+          runner: ["ubuntu-latest", "ubuntu-24.04", "ubuntu-20.04"]
```

Comment:
is it ok to remove the -arm?

Reply:
Yea, this is the default of `ubuntu-latest` so I'm making it match to remove some build variance.